pkg/blockdeviceclaim/v1alpha1/blockdeviceclaim.go

```diff
@@ -118,3 +121,30 @@ func (bdcl *BlockDeviceClaimList) GetBlockDeviceNamesByNode() map[string][]strin
 	}
 	return newNodeBDList
 }
+
+// RemoveFinalizersOnBDCList removes the given finalizer from the all BDC
+// belongs to provided label
+func RemoveFinalizersOnBDCList(namespace, label, finalizer string) error {
```

Comment:
IMO, better to take ListOption as an argument instead of passing the label -- will help in extra tuning. What is your thought?